### PR TITLE
✨ PRTL-747 Add data category - DNA methylation. Add tooltips for table heads

### PR DESCRIPTION
**Pull request by cheapsteak**

### CODE REVIEW

File: modules/node_modules/@ncigdc/containers/CaseTable.js [hunk 1]
```diff
@@ -72,17 +76,22 @@ export const CaseTableComponent = (props: TTableProps) => {
               <Th rowSpan="2">Primary Site</Th>
               <Th rowSpan="2">Gender</Th>
               <Th rowSpan="2">Files</Th>
-              <Th colSpan="6">Available Files per Data Category</Th>
+              <Th colSpan="7">Available Files per Data Category</Th>
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/containers/CaseTable.js]: seeing as you are doing this in the project table, might as well use `Object.keys(DATA_CATEGORIES).length` here :)

File: modules/node_modules/@ncigdc/utils/data/index.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ export const CATEGORY_MAP: TCategoryMap = {
   Exp: 'Transcriptome Profiling',
   SNV: 'Simple Nucleotide Variation',
   CNV: 'Copy Number Variation',
+  Meth: 'DNA Methylation',
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/utils/data/index.js]: Could this be built from the the one in constants.js?

Comment (cheapsteak) [inline modules/node_modules/@ncigdc/utils/data/index.js]: yup done :)

File: modules/node_modules/@ncigdc/containers/CaseTable.js (hunk 1 above)
Comment (cheapsteak) [inline modules/node_modules/@ncigdc/containers/CaseTable.js]: nice catch

File: modules/node_modules/@ncigdc/utils/data/index.js [hunk 3]
```diff
@@ -7,15 +9,7 @@ import type {
...
-export const CATEGORY_MAP: TCategoryMap = {
-  Seq: 'Raw Sequencing Data',
-  Exp: 'Transcriptome Profiling',
-  SNV: 'Simple Nucleotide Variation',
-  CNV: 'Copy Number Variation',
-  Clinical: 'Clinical',
-  Bio: 'Biospecimen',
-  DNA: 'DNA Methylation',
-};
+export const CATEGORY_MAP: TCategoryMap = _.fromPairs(DATA_CATEGORIES.map(c => [c.abbr, c.full]));
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/utils/data/index.js]: isn't `DATA_CATEGORIES` an object?

Comment (cheapsteak) [inline modules/node_modules/@ncigdc/utils/data/index.js]: 😱

Comment (cheapsteak) [inline modules/node_modules/@ncigdc/utils/data/index.js]: fixed

**Pull request merged**